qa/tasks/caasp.py

```diff
@@ -0,0 +1,100 @@
+'''
+Task that deploys a CAASP cluster on all the nodes
+Linter:
+    flake8 --max-line-length=100
+'''
+import logging
+from teuthology.misc import (
+    all_roles_of_type,
+    append_lines_to_file,
+    get_file
```

Comment:
(style) either move close parenthesis on this line, or add a comma after `get_file`

Reply:
Done